versions.gradle

```diff
@@ -1,27 +1,23 @@
 ext {
-
     // Project
     libVersion = "3.0.0"
     compileSdkVersion = 30
-    targetSdkVersion = 30
+    buildToolsVersion = "30.0.3"
     minSdkVersion = 14
+    targetSdkVersion = 30
 
-    // Gradle's
-    androidGradleVersion = '4.1.2'
-    ktlintGradleVersion = '10.0.0'
-    mavenGradleVersion = '2.1'
+    // Gradle
+    gradleVersion = '4.1.3'
+    ktlintVersion = '10.0.0'
 
     // JetBrains
-    kotlinVersion = '1.4.30'
+    kotlinVersion = '1.4.32'
     coroutinesVersion = '1.4.2'
 
-    // Android
-    materialVersion = '1.4.0-alpha01'
-
     // AndroidX
-    androidXAppCompatVersionSample = '1.3.0-rc01'
-    androidXAppCompatVersionCropper = '1.2.0'  // Used to avoid not release stable versions on the lib
```

Comment:
This changes broke the sample project, please run the sample app and CIs for testing, more on this on my PR comment.